laravel/etc/nginx/default.conf

```diff
@@ -0,0 +1,57 @@
+map $http_cloudfront_forwarded_proto $cloudfront_proto {
+  default "http";
+  https   "https";
+}
+
+map $http_cloudfront_forwarded_proto $cloudfront_https {
+  default "off";
+  https   "on";
+}
+
+map $http_cloudfront_forwarded_proto $strict_transport_security {
+  default "";
+  https   "max-age=31536000";
+}
+
+map $http_x_amzn_trace_id $processed_request_id {
+  default  "$http_x_amzn_trace_id";
+  ''       "$request_id";
+}
+
+map $uri $static_status_code {
+  include /etc/nginx/static_status_codes.conf;
+}
+
+map $host$uri $redirectdomain {
+  include /etc/nginx/redirects.conf;
+}
+
+map $request_uri $header_x_robots_tag {
+  include /etc/nginx/robots.conf;
+}
+
+include /etc/nginx/http.conf;
+
+server {
+  listen 8080;
+
+  root /data/public;
```

Comment:
If this is the only difference is there a way we can set it via a build arg or something?